lib/kennel/syncer.rb

```diff
@@ -8,7 +8,6 @@ class Syncer
     LINE_UP = "\e[1A\033[K" # go up and clear
 
     Plan = Struct.new(:noop?, :no_change, :create, :update, :delete, keyword_init: true)
-    Update = Struct.new(:update_log, keyword_init: true)
```

Comment:
Would prefer to keep this one since it allows for some extensibility. Returning _just_ an array of (rather odd nameless) structures makes it hard to add anything in future.

Reply:
prefer to keep the api simple until we need anything else
also "what was updated" seems like the obvious return value for an "update method"
adding an extra struct does not make that simpler
```
actions = syner.update.actions
```
it not much better 🤷

Reply:
I'm not arguing for keeping the "plan and update" level of wrapper. Feel free to get rid of that.

I am arguing for keeping the "actions" / "log" wrapper. i.e. don't just return an array, return something which responds to e.g. `:actions` to get that array.